### ISSUE TITLE
Investigate sortable columns for file upload table

### ISSUE THREAD

**Issue by marlo-longley** [ivplus/aggregator#500]
This question is arising out of #425 . From @ggeisler :

_I wonder if we can help the user who is checking the status of files using this table even more by making the table columns sortable? I don't know this for sure, but it seems possible that it could be helpful to be able to sort by the Content type or Records fields when using this table to check on something related to files._

Proposed sortable columns:
- Content Type
- Records

I might add "Status" too, if we go ahead with this.

<img width="1067" alt="158267545-0be5b290-f107-4b6f-b3dd-cddb298ef68d" src="https://user-images.githubusercontent.com/1328900/159545697-dc8e222b-130f-45bc-9cc8-d8c0846c081a.png">

**Comment by ggeisler** [comment]
I agree, @marlo-longley, sorting by Status would make sense as well, if we decide to do the extra work to add sorting. I'm unclear on whether that extra work is worth it, though, since I don't really know how often users use this table or what their specific use cases for doing so might be (e.g., I don't know how checking status of something in this table might relate to the Processing status info we display elsewhere).

**Comment by cbeer** [comment]
I'm wary of making this table sortable, because the files have a defined order that they must be processed in. Maybe we want filtering?

**Comment by ggeisler** [comment]
Well, again, not really knowing the use cases for this table, I'm not sure. I'm assuming this is a list of files in a stream that have completed processing and the user would want to go to the table to check on things like:

- Which files have the most records?
- Which files are the largest, file-size wise?
- I just want to see my marc files (or deletes) to assess something
- I know there are a bunch of invalid marc files, let me just focus on them

I have no idea if those are real use cases. The first two probably require sorting. The last two could work with either filtering or sorting. So filtering could be sufficient. (Doing nothing could be sufficient as well.) Maybe it's too early to worry about things like this and we could revisit after we get real users doing real work using the revised application and find out what their real use cases are.
